src/app/file-apploading/file-apploading.component.html

```diff
@@ -0,0 +1,7 @@
+<div class='flex'>
+   <input [hidden]='true' id='uploadFile' multiple type="file" (change)='aploadFile($event)'/>
```

Comment:
It's highly recommended to use double quotes for HTML attributes, but at least keep them consistent
`type="file"` `id='uploadFile'`